apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx

```diff
@@ -175,21 +188,34 @@ export const handleAttributeSettingsFormSubmit = (idpId: string, values: Identit
             outboundProvisioningRoles: [""]
         } as IdentityProviderRolesInterface
     ).then(() => {
-        dispatch(addAlert(
+        store.dispatch(addAlert(
             {
-                description: "Successfully updated role mapping configurations.",
+                description: I18n.instance.t("devPortal:components.idp.notifications." +
+                    "updateIDPRoleMappings.success.description"),
                 level: AlertLevels.SUCCESS,
-                message: "Update successful"
+                message: I18n.instance.t("devPortal:components.idp.notifications.updateIDPRoleMappings.success.message")
```

Comment:
Possible line length issue here. Please check.

Reply:
Not according to Webstorm.